packages/accordion/src/Accordion.ts

```diff
@@ -91,23 +87,22 @@ export class Accordion extends SizedMixin(SpectrumElement, {
         });
     }
 
-    private handleSlotchange(): void {
+    private handleSlotchange({
```

Comment:
@vjosyula @najikahalsema I sourced this technique from the button group which has a similar parent-child relationship with button. It passes the size value down to it's children during slot changes.